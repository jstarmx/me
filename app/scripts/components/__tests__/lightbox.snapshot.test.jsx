import React from 'react';
import renderer from 'react-test-renderer';

import Lightbox from '../lightbox';

it('renders a Lightbox component', () => {
  const component = renderer.create(<Lightbox />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
