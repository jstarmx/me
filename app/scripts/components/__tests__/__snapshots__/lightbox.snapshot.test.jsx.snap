// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Lightbox component 1`] = `<div />`;
